lib/linux_admin/network_interface/network_interface_rh.rb

```diff
@@ -0,0 +1,138 @@
+require 'ipaddr'
+require 'pathname'
+
+module LinuxAdmin
+  class NetworkInterfaceRH < NetworkInterface
+    IFACE_DIR = "/etc/sysconfig/network-scripts"
+
+    # @return [Hash<String, String>] Key value mappings in the interface file
+    attr_reader :interface_conf
```

Comment:
Should this be private?  I know you're using it for tests, but I'm not sure it makes sense in the public interface.  Even so, can you spell out `conf` to `config`.  Perhaps there's a better word than `conf`?